src/clj/game/cards/assets.clj

```diff
@@ -681,7 +697,9 @@
     :events {:corp-turn-begins ability}})
 
    "Melange Mining Corp."
-   {:abilities [{:cost [:click 3] :effect (effect (gain :credit 7)) :msg "gain 7 [Credits]"}]}
+   {:abilities [{:cost [:click 3] :effect (effect (do (gain :credit 7)
+                                                      (trigger-event state side :corp-spend-click (:cid card) 3)))
```

Comment:
Will check if I can make that work.  Now understand the issue with Melange.  thanks.